collect_app/src/test/java/org/odk/collect/android/support/Actions.kt

```diff
@@ -0,0 +1,56 @@
+package org.odk.collect.android.support
+
+import android.view.View
+import android.widget.NumberPicker
+import androidx.test.espresso.UiController
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.action.GeneralLocation
+import androidx.test.espresso.action.GeneralSwipeAction
+import androidx.test.espresso.action.Press
+import androidx.test.espresso.action.Swipe
+import androidx.test.espresso.matcher.ViewMatchers
+import org.hamcrest.Matcher
+
+object Actions {
+    fun scrollNumberPickerToValue(targetValue: Int): ViewAction {
```

Comment:
This could move to `ViewActions` in `test-shared` I think?

Reply:
RIght, done.